static/app/views/replays/list/replaysErroneousDeadRageCards.tsx

```diff
@@ -149,6 +149,11 @@ function CardTable({
     perPage: 3,
   });
 
+  const gridRows = new Array(Math.min(replays?.length ?? 0, 2))
+    .fill(' ')
+    .map(_ => '1fr')
+    .join(' ');
```

Comment:
there is still some kind of off-by-one problem
![SCR-20230726-lynq](https://github.com/getsentry/sentry/assets/187460/ba6ed616-c8a1-4844-8f67-39be686425ef)

Reply:
<img width="1207" alt="SCR-20230726-mcfc" src="https://github.com/getsentry/sentry/assets/56095982/cb072e80-097f-436f-a209-e91df3116603">
i believe it is fixed with newest commit :D